src/commands/deploy/deploy.ts

```diff
@@ -572,6 +575,7 @@ const bundleEdgeFunctions = async (options, command: BaseCommand) => {
   // eslint-disable-next-line n/prefer-global/process, unicorn/prefer-set-has
   const argv = process.argv.slice(2)
   const statusCb =
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
You could use this: https://github.com/netlify/cli/blob/e1372a3c228246c7feb9bf4f4e63dad547b2bfa9/src/utils/command-helpers.ts#L300